packages/dataviews/src/test/dataviews.tsx

```diff
@@ -258,6 +258,18 @@ describe( 'DataViews component', () => {
 
 		it( 'should trigger the onClickItem callback if isItemClickable returns true and title field is clicked', async () => {
 			const onClickItemCallback = jest.fn();
+			function LinkComponent( { item, ...props }: { item: Data } ) {
```

Comment:
@oandregal I updated the unit tests properly but I'm not really sure whether these tests are being run in Calypso and how to run them locally. Any idea?

Reply:
You can run them via `yarn run test-packages packages/dataviews`.